bin/helm_toolbox.sh

```diff
@@ -44,8 +44,17 @@ function upsert() {
     local tiller_version=$(helm version --server --short --tiller-connection-timeout $TIMEOUT 2> /dev/null | grep -Eo "v[0-9]+\.[0-9]+\..+")
     if [ "$helm_version" != "$tiller_version" ]; then
         info "Helm version: $helm_version, differs with tiller version: ${tiller_version:-'not installed'}"
-        info "Upgrarding tiller to $helm_version"
-        helm init --upgrade --force-upgrade --wait > $STDOUT
+        info "Upgrarding tiller to $helm_version"        
+        local tiller_serviceaccount=tiller
+        local tiller_serviceaccount_exists=$(kubectl get serviceaccount -n kube-system --ignore-not-found=true --request-timeout=${TIMEOUT}s -o name $tiller_serviceaccount 2> /dev/null)
+        local tiller_clusterrolebinding_exists=$(kubectl get clusterrolebinding --ignore-not-found=true --request-timeout=${TIMEOUT}s -o name tiller-cluster-rule 2> /dev/null)
```

Comment:
Shouldn't we only do this if RBAC enabled?

Reply:
Yeap, you're right.
I've used RBAC_ENABLED env for it.